src/System Application/App/ControlAddIns/src/SatisfactionSurvey/SatisfactionSurvey.ControlAddin.al

```diff
@@ -0,0 +1,123 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Integration;
+
+controladdin SatisfactionSurvey
+{
+    RequestedHeight = 485;
+    MinimumHeight = 485;
+    RequestedWidth = 300;
+    MinimumWidth = 200;
+    VerticalStretch = false;
+    VerticalShrink = false;
+    HorizontalStretch = true;
+    HorizontalShrink = true;
+
+    Scripts = 'ControlAddIns\src\WebPageViewer\js\WebPageViewerHelper.js',
+              'ControlAddIns\src\WebPageViewer\js\WebPageViewer.js';
+    StartupScript = 'ControlAddIns\src\WebPageViewer\js\Startup.js';
+    RecreateScript = 'ControlAddIns\src\WebPageViewer\js\Recreate.js';
+    RefreshScript = 'ControlAddIns\src\WebPageViewer\js\Refresh.js';
+    StyleSheets = 'ControlAddIns\src\WebPageViewer\stylesheets\WebPageViewer.css';
+    Images = 'ControlAddIns\src\WebPageViewer\images\Callback.html', 'ControlAddIns\src\WebPageViewer\images\Loader.gif';
```

Comment:
That seems hacky. We have an html document in an "Images" category, and the document only contains a single script tag for a callback - why not include it as just a script?

Reply:
I agree, however the Satisfaction Survey Add-in is just 1-1 converted from the legacy add-in. I don't know if this was done for a specific reason and changing it around would affect the behaviour.